File: backend/DataAccess/DaoManual/システム共有/tm_afctrlDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             コントロールマスタ
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tm_afctrlDto : DaEntityModelBase
    {
        public const string TABLE_NAME = "tm_afctrl";
        public string ctrlmaincd { get; set; }                    //コントロールメインコード
        public int ctrlsubcd { get; set; }                        //コントロールサブコード
        public string ctrlcd { get; set; }                        //コントロールコード
        public string itemnm { get; set; }                        //項目名称
        public EnumDataType datatype { get; set; }                //データ型
        public bool rangeflg { get; set; }                        //範囲フラグ
        public string? value1 { get; set; }                       //値１
        public string? value2 { get; set; }                       //値２
        public string? biko { get; set; }                         //備考
        public string upduserid { get; set; }                     //更新ユーザーID
        public DateTime upddttm { get; set; }                     //更新日時
    }
}

File: GJSBackend/BusinessService/Db/DaoManual/システム共有/tm_afctrlDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             コントロールマスタ
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Db
    Public Class tm_afctrlDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tm_afctrl"
        Public Property ctrlmaincd As String                    'コントロールメインコード
        Public Property ctrlsubcd As Integer                        'コントロールサブコード
        Public Property ctrlcd As String                        'コントロールコード
        Public Property itemnm As String                        '項目名称
        Public Property datatype As EnumDataType                'データ型
        Public Property rangeflg As Boolean                        '範囲フラグ
        Public Property value1 As String                       '値１
        Public Property value2 As String                       '値２
        Public Property biko As String                         '備考
        Public Property upduserid As String                     '更新ユーザーID
        Public Property upddttm As Date                     '更新日時
    End Class
End Namespace
